src/web/handlers/PanGestureHandler.ts

```diff
@@ -52,7 +52,7 @@ export default class PanGestureHandler extends GestureHandler {
   private lastY = 0;
 
   private activateAfterLongPress = 0;
-  private activationTimeout = 0;
+  private activationTimeout: ReturnType<typeof setTimeout> | undefined;
```

Comment:
Is this change in a scope of this PR?

Reply:
At the beginning it was, because I added `node` types into `tsconfig`. Later on I've removed them, as @j-piasecki suggested (now we have `@ts-ignore`, if you see better option - I'd be glad to hear it).

I brought back old types in [a2185c6](https://github.com/software-mansion/react-native-gesture-handler/pull/2852/commits/a2185c6b0456c3bb9089594093f8a8d6d733907f).